frontend/src/scenes/feature-flags/FeatureFlag.tsx

```diff
@@ -78,484 +94,1220 @@ export function FeatureFlag({ id }: { id?: string } = {}): JSX.Element {
-                                            fullWidth
-                                            checked={value}
+                                    <Field name="name" label="Description">
+                                        <LemonTextArea
+                                            className="ph-ignore-input"
+                                            data-attr="feature-flag-description"
+                                            placeholder="Adding a helpful description can ensure others know what this feature is for."
                                         />
-                                        <div className="text-muted text-sm pl-7">
-                                            If your feature flag is applied prior to an identify or authentication
-                                            event, use this to ensure that feature flags are not reset after a person is
-                                            identified. This ensures the experience for the anonymous person is carried
-                                            forward to the authenticated person.
-                                        </div>
-                                    </div>
-                                )}
-                            </Field>
-                        </Col>
-                        <Col span={12}>
-                            <FeatureFlagInstructions featureFlagKey={featureFlag.key || 'my-flag'} />
-                        </Col>
-                    </Row>
+                                    </Field>
+                                    <Field name="ensure_experience_continuity">
+                                        {({ value, onChange }) => (
+                                            <div className="border rounded p-4">
+                                                <LemonCheckbox
+                                                    id="continuity-checkbox"
+                                                    label="Persist flag across authentication steps"
+                                                    onChange={() => onChange(!value)}
+                                                    fullWidth
+                                                    checked={value}
+                                                />
+                                                <div className="text-muted text-sm pl-7">
+                                                    If your feature flag is applied prior to an identify or
+                                                    authentication event, use this to ensure that feature flags are not
+                                                    reset after a person is identified. This ensures the experience for
+                                                    the anonymous person is carried forward to the authenticated person.
+                                                    Currently supported for posthog-js only.
```

Comment:
The old copy is the 'right' one, since this is now supported everywhere else as well. (out of beta)